Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorThresholdEffect.py

```diff
@@ -2,10 +2,9 @@
 import os
 
 import ctk
-import vtk
 import qt
-
 import slicer
+import vtk
```

Comment:
Some issues seemingly go on related to ordering
```
Traceback (most recent call last):
  File "C:\S5R\Slicer-build\lib\Slicer-5.1\qt-scripted-modules\SegmentEditorEffects\SegmentEditorThresholdEffect.py", line 40, in __init__
    self.setupPreviewDisplay()
  File "C:\S5R\Slicer-build\lib\Slicer-5.1\qt-scripted-modules\SegmentEditorEffects\SegmentEditorThresholdEffect.py", line 648, in setupPreviewDisplay
    if not self.scriptedEffect.segmentationDisplayableInView(sliceWidget.mrmlSliceNode()):
TypeError: method requires a VTK object
qSlicerPythonCppAPI::instantiateClass  - [ "SegmentEditorThresholdEffect" ] - Failed to instantiate scripted pythonqt class "SegmentEditorThresholdEffect" 0x23b3d977090
```

Reply:
This ordering issue is likely what was described in the following comment:

https://github.com/Slicer/Slicer/issues/6484#issuecomment-1198453740

> you need to import vtk before you instantiate VTK-based classes.